Description: Fix bashism in data/m00
 Replace 'echo' with 'printf' which is more portable and
 reliable when printing strings with backslashes.
Author: Alessio Treglia <alessio@debian.org>
Bug-Debian: https://bugs.debian.org/772264
Forwarded: jag@dinisnoise.org
---
 data/m00 |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- din.orig/data/m00
+++ din/data/m00
@@ -2,4 +2,4 @@
 ACCEL=`xset q | grep accel | cut -d' ' -f5 |cut -d'/' -f1`
 THRESHOLD=`xset q | grep thres | cut -d' ' -f11`
 xset m 0 0
-echo "\033[1;30m<<< mouse parameters:`xset q | grep accel` >>> \033[0m"
+printf "\033[1;30m<<< mouse parameters:%s >>> \033[0m\n" "`xset q | grep accel`"
